app/src/main/java/com/google/samples/apps/sunflower/workers/SeedDatabaseWorker.kt

```diff
@@ -27,18 +27,18 @@ import com.google.samples.apps.sunflower.data.AppDatabase
 import com.google.samples.apps.sunflower.data.Plant
 import com.google.samples.apps.sunflower.utilities.PLANT_DATA_FILENAME
 
-class SeedDatabaseWorker(val context: Context, workerParams: WorkerParameters) : Worker(context, workerParams) {
+class SeedDatabaseWorker(context: Context, workerParams: WorkerParameters) : Worker(context, workerParams) {
     private val TAG = SeedDatabaseWorker::class.java.simpleName
 
     override fun doWork(): Worker.Result {
         val plantType = object : TypeToken<List<Plant>>() {}.type
         var jsonReader: JsonReader? = null
 
         return try {
-            val inputStream = context.assets.open(PLANT_DATA_FILENAME)
+            val inputStream = applicationContext.assets.open(PLANT_DATA_FILENAME)
```

Comment:
The whole `return` should be oriented to `Kotlin` style:

```
  applicationContext.assets.open("").use {
            JsonReader(it.reader()).use {
                
            }
        }
```

Reply:
That's outside the scope of this PR

Reply:
ok, however, this approach can be a little more affluent to read than Java style.

Reply:
In some cases could be useful to have a local variable to avoid the call to getApplicationContext() (exposed by Kotlin as property: applicationContext). But in this case seems fair to just use the property syntax.

LGTM

Reply:
@pfmaggi My *changes requested*  here is about using `use` which can safe `close` the `closeable` interfaces like `stream` in order to reduce line like `finally{close()}`.

Reply:
The optimise #201 for this.